### ISSUE TITLE
Certain Queries Produced Undefined Array Key Error

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/discoveryutils#229]
Testing in prod, staging and locally certain queries produce a 500 error. I think it may be related to how results the query matches leaving the summon API response to leave out the "StartPage" value the exception string below references. 

Example: "Burn it Down": articles/any?query=Burn+it+down&sid=library-main-summon
Example: "Pluralistic ignorance When similarity is interpreted as dissimilarity": articles/any?query=Pluralistic+ignorance+When+similarity+is+interpreted+as+dissimilarity&sid=library-main-summon

Error logged:

```ErrorException:
Warning: Undefined array key "StartPage"

  at classes/Summon/Record.php:63
  at Summon\Record->getFormattedDate()
     (classes/Summon/Response.php:56)
  at Summon\Response->getBriefResults()
     (src/Controller/ArticlesController.php:84)
  at App\Controller\ArticlesController->gather_data(object(Request), 'any', 'Burn it down')
     (src/Controller/BaseController.php:28)
  at App\Controller\BaseController->page(object(Logger), object(Request), 'any')
     (vendor/symfony/http-kernel/HttpKernel.php:159)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:81)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:201)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (public/index.php:25)```

**Comment by kevinreiss** [comment]
Just noting "new" allsearch doesn't have this error: https://allsearch.princeton.edu/?q=Pluralistic+ignorance+When+similarity+is+interpreted+as+dissimilarity so it's a weakness in this codebase.

**Comment by kevinreiss** [comment]
Closed by #230.
